allen-cell-animated/simularium-website
# Feature/focus button

Problem
=======
resolves #142 

Solution
========
added a focus button that toggles state, and a hotkey 

## Type of change
Please delete options that are not relevant.


Change summary:
---------------
* Updated the font (added instructions on how I do this to the [wiki](https://github.com/allen-cell-animated/simularium-website/wiki/Generate-a-font-with-svgs))

Steps to Verify:
----------------
1. npm start
2. select an agent
3. try to pan, won't allow you to really,  *NOTE: should we give the user some info here the first time they try this, ie, "turn off focus to pan away from selected agent?"
4. push focus button or "f"
5. try to pan, will be able to, and agent stays selected

Screenshots (optional):
-----------------------
<img width="330" alt="Screen Shot 2021-06-21 at 1 12 00 PM" src="https://user-images.githubusercontent.com/5170636/122822074-79798f80-d292-11eb-86bb-df53eef5bb27.png">
<img width="293" alt="Screen Shot 2021-06-21 at 1 12 13 PM" src="https://user-images.githubusercontent.com/5170636/122822083-7aaabc80-d292-11eb-82af-956f87e8f231.png">


Keyfiles (delete if not relevant):
-----------------------
1. [main file](https://github.com/allen-cell-animated/simularium-website/compare/feature/focus-button?expand=1#diff-642e7c0b8edb5c03031dbb5e9275f522e0d0ef25bf63e0d38fe5691de3842aab)

Thanks for contributing!
